.github/workflows/makefile.yml

```diff
@@ -0,0 +1,37 @@
+# Workflow for using the makefile to create containers and run tests
+name: Makefile CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-centos-v11_1_1:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the project
+      uses: actions/checkout@v3
+    - name: build another version using docker
+      run: make VERTICA_VERSION=11.1.1 OSTAG=centos
```

Comment:
why 11.1.1 and not the latest?

Reply:
I picked one at random and I figured the latest version would be tested by the developer already.  I probably should build ALL versions and test ALL versions.

Reply:
Yes or maybe for the 3 last releases depending on the strategy. So we can write in the README which Vertica version we support.